cape-ph/cape-cod
# Get to baseline of datalake demo with ResourceComponent architecture changes

# TO TEST
* tear down any currently deployed version of this stack.
* `pulumi up` this branch
* ensure there are no failures
* run through the datalake demo flow and ensure things work as they should.
